### ISSUE TITLE
Puppet run fails if $handle_database is false

### ISSUE THREAD

**Issue by adoom42** [anderssh/ash-netbox#10]
I manage PostgreSQL on my own, so I don't want the netbox module to do it.

When $handle_database is false, a Puppet run on the netbox server returns this error:
```
Error: Could not retrieve catalog from remote server: Error 500 on SERVER: Server Error: Could not find resource 'Class[Netbox::Database]' for relationship on 'Class[Netbox::Redis]' on node netbox.mydomain
```
It appears to be caused by this line:
https://github.com/anderssh/ash-netbox/blob/f887d1b490f1ee665df064d541bdbff9bc0b89fe/manifests/init.pp#L282

The following code from init.pp only instantiates `netbox::database` if $handle_database is true (good), but the dependency chain on line 282 expects it to exist:
```
  if $handle_database {
    class { 'netbox::database':
      database_name     => $database_name,
      database_user     => $database_user,
      database_password => $database_password,
      database_encoding => $database_encoding,
      database_locale   => $database_locale,
    }
  }
```
Manually removing `Class['netbox::database'] ->` from line 282 resolves the problem.  You may need to setup the dependency chain conditionally, leaving out `Class['netbox::database']` if $handle_database is false.

**Comment by anderssh** [comment]
Thanks for submitting an issue, I will have a look at it as soon as I got the chance!

**Comment by lukebigum** [comment]
I've fixed this in #14
